src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.js

```diff
@@ -206,33 +170,23 @@ describe.skip('VAOS Component: CanceledAppointmentsList', () => {
   });
 
   it('should show at home video appointment text', async () => {
-    const startDate = moment.utc();
-    const data = {
-      id: '1234',
-      currentStatus: 'CANCELLED BY CLINIC',
```

Comment:
What is the purpose of the `currentStatus` attribute? I don't see this attribute used in any vaos sources but I'm not sure if I missed anything so I kept it in the tests.